nssp/tests/test_patch.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from unittest.mock import patch as mock_patch, call
+from delphi_nssp.patch import patch
+import os
+import shutil
+from datetime import datetime, timedelta
+
+class TestPatchModule(unittest.TestCase):
```

Comment:
I think we need another test that given the patch params it will run the else condition, maybe in test_pull?

you can use the existing test to build the test with the parameters with source_dir and issue_date

Reply:
[Added](https://github.com/cmu-delphi/covidcast-indicators/pull/2000/files#diff-9bc87d511e1d29db37345fe99f5a82ad0aac79053f05ea57d303b18569fbfc62R25-R34).